Fix staking data

AurevoirXavier opened the pull request:
Query the um-migrated data with these keys:
```
λ subalfred storage-key --pallet Staking --item RingPool
0x5f3e4907f716ac89b6347d15ececedca1ab2669620e457d4fa08f305b84703ed

λ subalfred storage-key --pallet Staking --item KtonPool
0x5f3e4907f716ac89b6347d15ececedca00444b5e05d0712fd694bc34f31213d2

λ subalfred storage-key --pallet Staking --item ElapsedTime
0x5f3e4907f716ac89b6347d15ececedcad8552780d260417993181b12cc52f42e
```

boundless-forest commented on tool/state-processor/src/staking/mod.rs:
```diff
@@ -154,16 +154,21 @@ impl<S> Processor<S> {
-			b"Staking",
+		log::info!("set `DarwiniaStaking::RingPool` and `DarwiniaStaking::KtonPool`");
+		self.shell_state.insert_value(b"DarwiniaStaking", b"RingPool", "", ring_pool).insert_value(
+			b"DarwiniaStaking",
```
![image](https://user-images.githubusercontent.com/11801722/223956001-728e68f1-e755-48a4-a90e-8acde94062e9.png)

I think we need to update these constants  to make CI happy.

AurevoirXavier commented:
## Pangoro try runtime
```
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: current     `ring_pool(145099000000000000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: un-migrated `ring_pool(108074244177664000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: fixed       `ring_pool(253173244177664000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: current     `kton_pool(50348700000000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: un-migrated `kton_pool(116500000000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: fixed       `kton_pool(50465200000000000000)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: current     `elapsed_time(12192212101)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: un-migrated `elapsed_time(14691132009)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: genesis     `elapsed_time(11_516_352_020)`    
2023-03-09 16:06:22.707  INFO main pangoro_runtime::migration: fixed       `elapsed_time(15366992090)`
```
